### ISSUE TITLE
Update to git 2.17 breaks a little

### ISSUE THREAD

**Issue by slackhead** [olivierverdier/zsh-git-prompt#120]
The error message that is printed when not in a git repo has changed from:

fatal: Not a git repository to
fatal: not a git repository

Which causes gitstatus.py to output garbage results. Easy fix is to compare using .lower()

**Comment by fam4r** [comment]
Please see here: https://github.com/olivierverdier/zsh-git-prompt/issues/103

**Comment by juanibiapina** [comment]
A better option is to compare the status code:

```
if gitsym.returncode != 0:
        sys.exit(0)
```

**Comment by fam4r** [comment]
Yes, that has been discussed in #103 and there's a PR in #119 , just in case you want to join the thread :smile:
